src/components/BeforeLogin/index.tsx

```diff
@@ -5,9 +5,15 @@ const BeforeLogin = (): ReactElement => {
 
     return (
         <div>
+            <h4>
+                <center>
+                    Logge dich hier mit deinem B-Side Account ein!
+                </center>
+            </h4>
             <p>
-                The CMS is currently not connected to our user management.<br />
-                Please contact the IT team for access.
+                Falls dir beim Login eine Fehlermeldung angezeigt wird, fehlen dir
+                die notwendigen Rechte für den Zugriff auf das Backend unserer Webseite.
+                Wende dich in dem Fall bitte bei Mattermost an das IT-Team.
```

Comment:
Gibt's da was neues? Hab unten in den Änderungen nichts ausmachen können 😊

Reply:
Beim SSO-Provider muss noch ein neuer Login-Flow eingestellt werden, der die Berechtigungen zum Login prüft. Damit müssen wir die Prüfung nicht hier in der Software machen.

Der Login-Flow gibt aber keine benutzerdefinierte Fehlermeldung. Also im Sinne "dir fehlt die Berechtigung zum Login bei Payload". Sondern es kommt einfach die gleiche Meldung wie bei falschen Passwort oder deaktiviertem Account. Daher hier die etwas längere Erklärung.